test/kwtool/crawlers/upload_parser_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule UploadParserTest do
+  use Kwtool.DataCase, async: true
+
+  alias Kwtool.Crawlers.UploadParser
+
+  describe "load_from_file/1" do
+    test "returns a list of keyword when uploads a valid CSV file" do
```

Comment:
Minor preference over phrasing using `given` as it's less verbose and remove the need to conjugate verbs:

```suggestion
    test "returns a list of keyword given a valid CSV file" do
```

Reply:
Updated! Thanks 👍